src/diffusers/pipelines/flux/pipeline_flux_controlnet.py

```diff
@@ -512,7 +526,7 @@ def prepare_latents(
         shape = (batch_size, num_channels_latents, height, width)
 
         if latents is not None:
-            latent_image_ids = self._prepare_latent_image_ids(batch_size, height, width, device, dtype)
+            latent_image_ids = self._prepare_latent_image_ids(batch_size, height // 2, width // 2, device, dtype)
```

Comment:
I can recollect two PRs here:

* https://github.com/huggingface/diffusers/pull/9711
* https://github.com/huggingface/diffusers/pull/9929

Gently pinging @DN6

Reply:
Oh, do you mean I am reverting a previous change that was necessary for something? I have been trying to pass `latents` for testing purposes but it always fails at this point unless the height and width are correctly adjusted -- it makes sense to me when comparing with other pipelines and doing rough calculation with tensor shapes, but LMK if this is wrong

Reply:
Not this is fine. We missed scaling the height and width when passing in latents. We do it here
https://github.com/huggingface/diffusers/blob/e564abe292750b7d2eef07f2b49ea2056df391ab/src/diffusers/pipelines/flux/pipeline_flux.py#L528

Reply:
Oh okay. Then I'll update the slices and merge once CI is green